server/app/util/filter.py

```diff
@@ -0,0 +1,64 @@
+class QueryStringError(Exception):
+    pass
+
+
+def _convert_variable(datatype, variable):
+    """
+    Convert variable to number (float/int)
+    Used for dataset metadata and for query string
+    :param datatype: type to convert to
+    :param variable: value of variable
+    :return: converted variable
+    :raises: ValueError
+    """
+    try:
+        if variable and datatype == "int":
```

Comment:
doesn't this break down if variable is 0?

Reply:
yes it does, good catch. I'll fix it